azext_iot/operations/hub.py

```diff
@@ -1284,8 +1284,10 @@ def iot_device_upload_file(cmd, device_id, file_path, content_type, hub_name=Non
         raise CLIError(e)
 
 
+# TODO: Shortcut for PnP
 # pylint: disable=too-many-locals
-def iot_hub_monitor_events(cmd, hub_name=None, device_id=None, consumer_group='$Default', timeout=300,
+def iot_hub_monitor_events(cmd, interface=None, pnp_context=None,
```

Comment:
The pnp_context argument at this layer was part of a private preview shortcut pattern. pnp_context shouldn't show up at the top level (by top level I mean the function associated with a command). Ideally a user running `az iot hub monitor-events` has no knowledge of pnp or a pnp_context, it's a less optimal user experience having it here.

Reply:
Consider a lower level "private" function that intakes pnp_context (among the other elements). BTW some improvements  to event monitoring setup is in work in the iot central PR.

Reply:
Same idea for `interface`.

Reply:
remove pnp_context and interface, create new private method